jenkins/ci.suse.de/cloud-mediacheck.yaml

```diff
@@ -50,8 +50,14 @@
           # fetch the latest automation updates
           ${automationrepo}/scripts/jenkins/update_automation
 
+          function cleanup() {
+              # Cleanup isos from old builds
+              rm -f scripts/release-mgmt/*Media1.iso
+          }
+
           function jtsync_trap() {
             $jtsync --ci suse --matrix ${JOB_BASE_NAME},${project},${BUILD_NUMBER} 1 || :
+            cleanup
```

Comment:
The trap will not be called for 'MANUALTRIGGER' jobs - possibly just have jenkins delete the workspace dir